app/services/data_exports/aggregate_metadata_file_generator.rb

```diff
@@ -0,0 +1,77 @@
+require 'csv'
+
+module DataExports
+  # Helper class for aggregating metadata from individual transcriptions
+  # within a group/workflow/project into a single csv file
+  class AggregateMetadataFileGenerator
+    class << self
+      # Public: add metadata csv file to group folder
+      def generate_group_file(transcriptions, output_folder)
+        @metadata_file = File.join(output_folder, 'transcriptions_metadata.csv')
```

Comment:
be careful with class instance vars that mutate state, especially if this will be called via threaded code.

Reply:
Yeah, better that I dont use class instance vars. I will update.